kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CaseRuntimeDataServiceIntegrationTest.java

```diff
@@ -554,6 +554,35 @@ public void testGetCaseInstancesWithData() {
         assertEquals("first case started", instance.getCaseFile().getData().get("s"));
     }
 
+    @Test
+    public void testByPassAuthCaseTasksAssignedAsPotentialOwner() throws Exception {
```

Comment:
I would rename it to something more meaningful with regards to the fix, in this case, when the user sent in the request is null or empty
```suggestion
    public void testByPassAuthCaseTasksAssignedAsPotentialOwnerWithEmptyUser() throws Exception {
```